core/src/avm2/globals/flash/display/graphics.rs

```diff
@@ -1054,10 +1059,20 @@ fn draw_triangle_internal((a, b, c): Triangle, drawing: &mut Drawing, culling: T
             drawing.draw_command(DrawCommand::LineTo(a));
         }
         TriangleCulling::Positive => {
-            panic!("Positive culling not implemented")
+            avm2_stub_method!(
+                activation,
+                "flash.display.Graphics",
+                "drawTriangles",
+                "Positive culling not implemented"
```

Comment:
The way this stub message will appear is `Encountered stub: AVM2 flash.display.Graphics.drawTriangles() Positive culling not implemented`. I think it's better to change this to `"with positive culling"` rather than "Positive culling not implemented" (same with negative culling).

Reply:
Updated.